test/unit/spec/v2/view-builder/views/EnrollPollOktaVerifyView_spec.js

```diff
@@ -62,4 +66,23 @@ describe('v2/view-builder/views/ov/EnrollPollOktaVerifyView', function() {
     expect(testContext.view.options.appState.trigger).not.toHaveBeenCalled();
     expect(testContext.view.$('.qrcode').length).toBe(1);
   });
+
+  it('switches to select enroll method form when on mobile', function(done) {
+    MockUtil.mockIntrospect(done, xhrAuthenticatorEnrollOktaVerifyQr, idxResp => {
```

Comment:
@aarongranick-okta  the issue I had earlier was that I was testing for something async in the assertion function here and I ding have to. 

Verified that the test fails on branch with custom render for baseController. 

<img width="1272" alt="Screen Shot 2021-08-11 at 9 57 42 AM" src="https://user-images.githubusercontent.com/17267130/129073376-13a11623-3171-4123-87e8-cf6331f8e31d.png">